.jshintrc

```diff
@@ -9,6 +9,8 @@
   "strict": "global",
   "undef": true,
   "unused": true,
+  "esversion": 6,
```

Comment:
I'm just concerned that while this silences warnings for the toolchain, it also potentially suppresses warnings about the actual client code.